### DISCUSSION TITLE
python syntax for while-loop

### DISCUSSION THREAD

**Discussion by AjitChandravanshi001** [AjitChandravanshi001/expert-octo-winner#4] [Q&A] [answered]
write a code for python-while loop

**Comment by Akarshjha03** [comment, marked as answer]
# Initialize the counter variable
counter = 1

# Start the while loop
while counter <= 10:
    print(counter)
    counter += 1  # Increment the counter by 1

**Comment by amlesh123632** [comment]
# Initialize the variable
num = 1

# Start the while loop
while num <= 5:
    print("Number:", num)
    num += 1  # Increment the variable
